### PR TITLE
Fix/review

### PR DESCRIPTION
**Pull request by xalbertoisorna**
Hi Pavel, made a few changes after our discussion yesterday. 
once merged I'll make the branch available in upstream.

### CODE REVIEW

File: README.rst
Comment (uvvpavel) [inline README.rst:1]: I'm not sure if all this belongs in the top-level readme

File: README.rst [hunk 1]
```diff
@@ -18,6 +18,50 @@ Examples
 - **take_picture_downsample** : takes a picture from an input size of 680x480 or 1280x960 and downsample it down to 160x120. 
 
 
+How to Use
+----------
+
+.. code-block:: c
+
+    // ---- main.xc
```
Comment (uvvpavel) [inline README.rst]: code-block:: xc 🤣

File: lib_camera/api/camera_utils.h [hunk 2]
```diff
@@ -108,5 +109,21 @@ void io_write_bmp_file(
...
+ * Same as standard `memcpy()` except for an extra constraint that both `dst and `src` must be
+ * word-aligned addresses.
+ *
+ * @param[out]  dst   Destination address
+ * @param[in]   src   Source address
+ * @param[in]   bytes Number of bytes to copy
+ *
+ * @ingroup util_macros
+ */
+void vpu_memcpy(
```
Comment (uvvpavel) [inline lib_camera/api/camera_utils.h]: If we want to keep using it, it should have a different name as I'm planning to rename the one in lib_xcore_math as well. Could also remove from the public api, don't feel strongly about it tho

File: lib_camera/src/sensors/imx219.cpp [hunk 3]
```diff
@@ -120,9 +129,9 @@ i2c_table_t IMX219::get_pxl_fmt_table() {
   static i2c_line_t format_regs[3];
   static i2c_table_t format_table = {format_regs, 3};
   uint16_t val = 0;
-  if(pix_fmt == MIPI_DT_RAW8) {
+  if(pix_fmt == SENSOR_MIPI_RAW8) {
     val = 0x08;
-  } else if(pix_fmt == MIPI_DT_RAW10) {
+  } else if(pix_fmt == SENSOR_MIPI_RAW10) {
     val = 0x0a;
```
Comment (uvvpavel) [inline lib_camera/src/sensors/imx219.cpp]: should we remove raw10?

**Review by uvvpavel — CHANGES_REQUESTED** [review]
few comments/ suggestions

File: README.rst
Comment (xalbertoisorna) [reply README.rst:1]: maybe to be placed in quickstart? is a good point actually, Ill ask tom

File: lib_camera/api/camera_utils.h (hunk 2 above)
Comment (xalbertoisorna) [reply lib_camera/api/camera_utils.h]: - opt_memcpy
- xmemcpy
- cpymem
- fast_memcpy
- funny_memcpy
- aligned_memcpy

File: README.rst
Comment (xalbertoisorna) [reply README.rst:1]: @uvvpavel removed until clarification

File: lib_camera/api/camera_utils.h (hunk 2 above)
Comment (xalbertoisorna) [reply lib_camera/api/camera_utils.h]: the winner is: xmemcpy

**Review by uvvpavel — APPROVED** [review]
Looks great

**Pull request merged**